UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Extensions/FluentValidationResultExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text.Json;
+using FluentValidation.Results;
+using UKHO.ExternalNotificationService.Common.Models.Response;
+
+namespace UKHO.ExternalNotificationService.API.Extensions
+{
+    public static class FluentValidationResultExtensions
+    {
+        public static bool HasBadRequestErrors(this ValidationResult validationResult, out List<Error> badRequestErrors)
+        {
+            return HasErrors(validationResult, HttpStatusCode.BadRequest, out badRequestErrors);
+        }
+
+        private static bool HasErrors(this ValidationResult validationResult, HttpStatusCode errorCode, out List<Error> errors)
+        {
+            errors = new List<Error>();
+            if (validationResult.Errors.Any(e => e.ErrorCode.Equals(errorCode.ToString())))
+            {
+                errors = validationResult.Errors.Where(e => e.ErrorCode.Equals(errorCode.ToString()))
+                                    .Select(f => new Error { Source = JsonNamingPolicy.CamelCase.ConvertName(f.PropertyName), Description = f.ToString() })
+                                    .ToList();
+                return true;
+            }
+            return false;
```

Comment:
This line has no test covering this.

Reply:
We don't have that condition right now, this PBI we covered 202 and 400 are the response codes.
Once we have covered any other errors, we'll cover this.